catalog/ui/search-ui/search-endpoint/src/test/java/org/codice/ddf/ui/searchui/query/service/WorkspaceServiceTest.java

```diff
@@ -32,7 +32,7 @@ public void test() throws Exception {
         //        map1.put("data", data1);
         //        map1.put("extra", extra1);
         //        String json = jsonContext.getGenerator().generate(map1);
-        JSONContext.Client jsonContext = new JacksonJSONContextClient();
+        JSONContext.Client jsonContext = new Jackson1JSONContextClient();
```

Comment:
I'm actually questioning the validity of this test at all. I was in a rush to make things compile / run and ignore the fact that there are no assertions.  I think it deserves some more attention to either be removed completely or updated to be an actual unit test.  Ticket coming.